docs/source/install/install.rst

```diff
@@ -395,6 +349,7 @@ The following browsers are unsupported:
 * Chrome < 13
 * Opera (any): CSS issues, but execution might work
 * Internet Explorer < 10
+* Internet Explorer ≥ 10 (same as Opera)
 
 The following specific combinations are known **NOT** to work:
```

Comment:
I disagree. I can remove the 0.12 note, but Safari still rejects websocket connections to self-signed certs.